### PR TITLE
slides: update from conn to sock

### PR DESCRIPTION
**Pull request by OlegHahm**
Update slide text and images to sock.

### CODE REVIEW

File: slides/slides.md [hunk 1]
```diff
@@ -268,23 +268,23 @@ USEMODULE += auto_init_gnrc_netif
...
+    - `sock_tcp` (TCP)
     - ...
 * both IPv4 and IPv6 supported
 
 ## Task 6.1 -- Use UDP for messaging
 * Go to `task-06` directory `cd ../task-06`
-* Note the addition of `gnrc_conn_udp` to Makefile
-* `udp.c` utilizes `conn_udp_sendto()` and `conn_udp_recvfrom()` to exchange UDP packets
+* Note the addition of `gnrc_sock_udp` to Makefile
+* `udp.c` utilizes `sock_udp_sendto()` and `sock_udp_recvfrom()` to exchange UDP packets
```
Comment (miri64) [inline slides/slides.md]: `sock_udp_send()` / `sock_udp_recv()`

**Review by miri64 — DISMISSED** [review]

**Comment by miri64** [comment]
Can you please also update https://github.com/RIOT-OS/Tutorials/blob/master/overview-net.png with your changes in [your changes of the corresponding SVG](https://github.com/RIOT-OS/Tutorials/pull/37/files#diff-2499fe47a0b410bcc911fe436a9d6871)

**Comment by miri64** [comment]
Fixes #41

**Comment by miri64** [comment]
Rebased to current master as #59 was merged. @emmanuelsearch can you have a look maybe?

**Comment by emmanuelsearch** [comment]
looks good.

**Review by emmanuelsearch — APPROVED** [review]

**Comment by miri64** [comment]
Rebased to current master to trigger Travis build.

**Pull request merged**